### ISSUE TITLE
qc_diag error

### ISSUE THREAD

**Issue by danish1162** [bkerler/edl#273]
hi 

/usr/lib/python3/dist-packages/pkg_resources/__init__.py:116: PkgResourcesDeprecationWarning: 0.1.39ubuntu1 is an invalid version and will not be supported in a future release
  warnings.warn(
Traceback (most recent call last):
  File "/usr/local/bin/qc_diag.py", line 4, in <module>
    __import__('pkg_resources').run_script('edlclient==3.60', 'qc_diag.py')
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3267, in <module>
    def _initialize_master_working_set():
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3241, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3279, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 573, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 891, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 777, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'edlclient==3.60' distribution was not found and is required by the application

**Comment by laminarturbulent** [comment]
I had this issue too. Using this part of the install guide: https://github.com/bkerler/edl#linux-debianubuntumintetc

Ran these commands in the edl terminal and it fixed it:
```
git submodule update --init --recursive
python setup.py build
sudo python setup.py install
```

**Comment by laminarturbulent** [comment]
Also you should run `git pull` in the edl directory to update it since the version included in [the Re LiveDVD](https://github.com/bkerler/edl#use-re-livedvd-everything-ready-to-go-based-on-ubuntu) hasn't been updated to fix #233 (skip userdata doesn't skip userdata) yet.
